### PR TITLE
Wires site_is_atomic from the reader endpoint.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Integrates the changes from https://github.com/wordpress-mobile/WordPressKit-iOS/pull/240, so that we can also integrate the changes in D41085 into the App.

Bottom line is: the reader endpoint will now return a `site_is_atomic` flag in posts from atomic sites.  This PR wires that new flag into WPiOS.

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/13823

## To test:

1. Follow a private atomic site in the reader
2. Publish a post in that site, including an image just to make it more interesting.
3. Go to Reader > Followed Sites > Managed  > Pick your Atomic Private site.

Check that posts and images are visible.

## PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/Models/ReaderPost.m [hunk 1]
```diff
@@ -67,11 +68,7 @@ - (BOOL)isCrossPost
 
 - (BOOL)isAtomic
 {
-    // TODO: This is temporary until we start parsing a new value from the reader endpoint.
-    //
-    // Issue: https://git.io/JvNAN
-    //
-    return false;
+    return self.isBlogAtomic;
```
Comment (aerych) [inline WordPress/Classes/Models/ReaderPost.m:71]: Would it make sense to update `ReaderPostContentProvider.h` to change `isAtomic` to `isBlogAtomic` and then update `MediaHost+ReaderPostContentProvider.swift` accordingly and then remove this `isAtomic` method completely?

Comment (jleandroperez) [reply WordPress/Classes/Models/ReaderPost.m:71]: +1 on this one!

**Comment by diegoreymendez** [comment]
Aight folks  thanks.  I'm merging and tackling anything else that's missing separately.

**Comment by diegoreymendez** [comment]
Ugh, apologies I'm a bit tired from the week and didn't realize there was a comment.  I'll consider that for a new PR.

**Comment by jleandroperez** [comment]
No worries @diegoreymendez, it was a nice to have / cosmetic change, the PR itself is solid!

**Pull request merged**